server/handlers/genomic_annotations_test_data.go

```diff
@@ -0,0 +1,232 @@
+package handlers
```

Comment:
Same comment as another: status check implementation should go in its own package (suggested server/status).

Reply:
Agreed